### PR TITLE
.travis.yml: Create travis config file

### LINKED ISSUE #23: Adding CI
We should add Travis/Circle CI

### PR DESCRIPTION
**Pull request by hemangsk**
Closes #23

### CODE REVIEW

**Comment by hemangsk** [comment]
Lets add travis! 🎉

**Comment by sils** [comment]
can you force push to trigger a build? Activated travis now

File: script/cibuild.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
+bundle exec jekyll build
```
Comment (sils) [inline script/cibuild.sh]: you don't seem to use this

**Comment by hemangsk** [comment]
Ready for review 👍

**Comment by sils** [comment]
ack https://github.com/viper-development/viper-development.github.io/pull/38/commits/49e3896711a3ff038f51a2f0dc411836496b3ea4

**Pull request merged**